contracts/cw20-stake/src/contract.rs

```diff
@@ -28,7 +28,7 @@ pub use cw20_base::contract::{
     execute_upload_logo, query_balance, query_download_logo, query_marketing_info, query_minter,
     query_token_info,
 };
-pub use cw20_base::enumerable::{query_all_accounts, query_all_allowances};
```

Comment:
Why is this part of the diff?

Reply:
Because between versions 0.14->0.15 `query_all_allowances` has been removed.
https://github.com/CosmWasm/cw-plus/commit/15f929a542e0ec03d1ce01559df8a595447cc55e

Though those `pub use`s doesn't make sense in the first place, since they are not used anywhere.